python/tests/sql_aggregates/test_array_agg.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+from feldera import PipelineBuilder, Pipeline
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data = [{"insert":{"id": 0, "c1": 5, "c2": 2, "c3": None, "c4": 4, "c5": 5, "c6": 6, "c7": None, "c8": 8}},
+                       {"insert":{"id": 1,"c1": 4, "c2": 3, "c3": 4, "c4": 6, "c5": 2, "c6": 3, "c7": 4, "c8": 2}},
+                       {"insert":{"id": 0, "c1": 1, "c2": 2, "c3": 3, "c4": 4, "c5": 5, "c6": 6, "c7": None, "c8": 8}},
+                       {"insert":{"id": 1,"c1": None, "c2": 20, "c3": 30, "c4": 40, "c5": None, "c6": 60, "c7": 70, "c8": 80}}]       
+        return super().setUp()
+      
+    def execute_query(self, pipeline_name, expected_data, table_name, view_query):
```

Comment:
this function should only exist in the base class, you can remove it from all other subclasses.
it's supposed to be identical.